api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java

```diff
@@ -131,6 +132,12 @@ public class DataSetServiceImpl implements DataSetService, GaugeDataCollector {
           + "\nAND UPPER(domain) = %s";
   private static final String LIMIT_20 = " LIMIT 20";
   private static final String PERSON_ID_COLUMN_NAME = "PERSON_ID";
+
+  // RStudio has a line length limit of 4096 characters.  Pasting lines longer than this will
+  // FAIL SILENTLY, CAUSING WRONG RESULTS, so it's critical to produce output shorter than this.
+  // https://github.com/rstudio/rstudio/issues/14420
+  private static final int RSTUDIO_LINE_LENGTH_MINUS_BUFFER = 4000;
```

Comment:
This feels like a reasonable margin of safety to me.  Happy to decrease it, if anyone wants more safety here

Reply:
4000 is bit too close to the limit. I would prefer to have a much smaller for both human readable and large buffer. 
Also, a self reminder: Check SAS size limit

Reply:
I'll make it 1000 for the first iteration